### DISCUSSION TITLE
Mohonn bantuannyaaa Puhh

### DISCUSSION THREAD

**Discussion by shamhasan** [deaafrizal/forum#38] [Troubleshooting Room]
![Screenshot (118)](https://github.com/deaaprizal/forum/assets/139510525/87051c9c-cb22-4c79-91f9-693a7a94c401)

gan, kenapa saya tidak bisa menuliskan baris kode berikut :
var itemText = toDoEntryBox.value;

melainkan Vscode otomatis membenarkan antaranya :
a. var itemText = toDoEntryBox.ariaValueMax;
b. var itemText = toDoEntryBox.ariaValueMin;
c. var itemText = toDoEntryBox.ariaValueNow;
d. var itemText = toDoEntryBox.ariaValueMax;
e. var itemText = toDoEntryBox.ariaValueText;
f. var itemText = toDoEntryBox.nodeValue;

Adakah yang faham dengan masalah saya dan solusi nya seperti apa ya kira kira.
terimakasih gan.

**Comment by deaafrizal** [comment]
autocomplete nya coba di matiin dulu kalau dari extension vscode, atau cara ngakalinnya pencet spasi spasi dulu aja terus delete 1 per 1 hurufnya ntar bisa kok.
